deployment_and_ssl_doc.md

```diff
@@ -107,10 +134,13 @@ word `exit`.
 
 ## Create an SSL Certificate
 
-For this step you will need to fork the Repo that can be found at this url
+Before creating an SSL certificate you must have a domain set up that you want
```

Comment:
I would possibly put this at the top with the list of things that this README expects the user to have before starting but this is not a blocker, just a thought

Reply:
That's a good point. I'll keep it here as a reminder but put a "before you get started" section at the top of the README.